server/ormconfig.ts

```diff
@@ -0,0 +1,16 @@
+import dotenv from 'dotenv';
+import { PostgresConnectionOptions } from 'typeorm/driver/postgres/PostgresConnectionOptions';
+dotenv.config();
+
+export default {
+  url: process.env.PG_DATABASE_URL,
+  type: 'postgres',
+  entities: [__dirname + '/src/core/**/*.entity{.ts,.js}'],
```

Comment:
@gitstart-twenty Should be `coreV2` here, and `userv2` should be placed in `coreV2` too